packages/playground/src/portal/twin_view.vue

```diff
@@ -52,16 +63,47 @@
 import { generatePublicKey } from "@threefold/rmb_direct_client";
 import { onMounted, ref } from "vue";
 
+import router from "../router";
 import { useProfileManager } from "../stores";
+import type { Farm } from "../types";
+import { getFarms } from "../utils/get_farms";
 import { getGrid } from "../utils/grid";
-
 const profileManager = useProfileManager();
 
 const editingTwin = ref(false);
 const relay = ref(profileManager.profile?.relay || "");
 const updateRelay = ref(false);
 const errorMsg = ref("");
+const openVotePopup = ref(false);
+const numberOfActiveProposals = ref(0);
+const userFarms = ref<Farm[]>();
+onMounted(async () => {
+  const profile = profileManager.profile!;
+  const grid = await getGrid(profile);
+  if (grid) {
+    userFarms.value = await getFarms(grid, { ownedBy: profile.twinId }, {});
+    if (userFarms.value.length > 0) {
+      const proposals = grid?.dao.get();
+      const userFarmId = userFarms.value.map(farm => farm.farmID);
+      const activeProposals = (await proposals)?.active;
+      numberOfActiveProposals.value = activeProposals ? activeProposals.length : 0;
+      if (numberOfActiveProposals.value > 0) {
+        const activeProposalsUserHasVotedOn = activeProposals.filter(
+          proposal =>
+            proposal.ayes.filter(aye => userFarmId.includes(aye.farmId)) ||
+            proposal.nayes.filter(naye => userFarmId.includes(naye.farmId)),
+        );
+        if (activeProposalsUserHasVotedOn.length != numberOfActiveProposals.value) {
+          openVotePopup.value = true;
```

Comment:
before showing the popup, please update the `numberOfActiveProposals` to be the number of the proposals he should vote on only